bin/utils.js

```diff
@@ -0,0 +1,303 @@
+/*
+ * ISC License (ISC)
+ * Copyright (c) 2018 aeternity developers
+ *
+ *  Permission to use, copy, modify, and/or distribute this software for any
+ *  purpose with or without fee is hereby granted, provided that the above
+ *  copyright notice and this permission notice appear in all copies.
+ *
+ *  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+ *  REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+ *  AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+ *  INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+ *  LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+ *  OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+ *  PERFORMANCE OF THIS SOFTWARE.
+ */
+'use strict'
+
+require = require('esm')(module/*, options*/) //use to handle es6 import/export
+const path = require('path')
+const fs = require('fs')
+const prompt = require('prompt')
+
+const {default: Cli} = require('../es/ae/cli')
+const Crypto = require('../es/utils/crypto')
+
+// HAST TYPES
+const HASH_TYPES = {
+  transaction: 'th',
+  block: 'bh',
+  signature: 'sg',
+  account: 'ak',
+  stateHash: 'bs'
+}
+
+// The `prompt` library provides concealed input of passwords.
+const PROMPT_SCHEMA = {
+  properties: {
+    password: {
+      type: 'string',
+      description: 'Enter your password',
+      hidden: true,
+      required: false,
+      replace: '*',
+      conform: function () {
+        return true
+      }
+    }
+  }
+}
+
+// FILE I/O
+const writeFile = (name, data) => {
+  try {
+    fs.writeFileSync(
+      name,
+      data
+    )
+    return true
+  } catch (e) {
+    printError('WRITE FILE ERROR: ' + e)
+    process.exit(1)
+  }
+}
+
+const readFile = (path, encoding = null) => {
+  try {
+    return fs.readFileSync(
+      path,
+      encoding
+    )
+  } catch (e) {
+    switch (e.code) {
+      case 'ENOENT':
+        printError('READ FILE ERROR: ' + 'File not found')
+        break
+      default:
+        printError('READ FILE ERROR: ' + e)
+        break
+    }
+    process.exit(1)
+  }
+}
+
+const readJSONFile = (filePath) => {
+  try {
+    return require(
+      path.resolve(process.cwd(), filePath)
+    )
+  } catch (e) {
+    printError('READ FILE ERROR: ' + 'File not found')
+    process.exit(1)
+  }
+}
+
+// CONSOLE PRINT HELPERS
+const print = (msg, obj = '') => console.log(msg, obj)
+
+const printError = (msg) => console.log(msg)
+
+const printConfig = ({host}) => {
+  console.log('WALLET_PUB___________' + process.env['WALLET_PUB'])
+  console.log('EPOCH_URL___________' + host)
+}
+
+const printBlock = (block) => {
+  console
+    .log(`Block hash____________________ ${block.hash}
```

Comment:
Inconsistency: Here, you are using multiline strings, but below, multiple print statements. Nothing major though.